x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -216,27 +238,41 @@ export const FieldsControls: FC<FieldsControlsProps> = ({ fieldConfig, onChange,
   );
 
   return (
-    <EuiFlexGroup alignItems={'center'} responsive={true} wrap={true} gutterSize={'m'}>
-      <EuiFlexItem grow={false} css={{ width: '200px' }}>
-        <FunctionPicker value={fieldConfig.fn} onChange={(v) => onChangeFn('fn', v)} />
-      </EuiFlexItem>
-      <EuiFlexItem grow={true} css={selectControlCss}>
-        <MetricFieldSelector
-          value={fieldConfig.metricField!}
-          onChange={(v) => onChangeFn('metricField', v)}
-        />
-      </EuiFlexItem>
-      {splitFieldsOptions.length > 0 ? (
+    <FieldStatsFlyoutProvider
```

Comment:
Is it possible to automatically close the flyout when the user makes a selection in the combo box? Thinking it might be better to close it once a selection has been made. What do you think?

Reply:
As discussed offline, I also get the pointer cursor anywhere over the flyout which can lead to some odd interaction when clicking inside the flyout. Can you reproduce this?

Reply:
> As discussed offline, I also get the pointer cursor anywhere over the flyout which can lead to some odd interaction when clicking inside the flyout. Can you reproduce this?

This happened because of the `EuiAccordion` trigger. I replaced it with a custom collapsible control in 
https://github.com/elastic/kibana/pull/155177/commits/38e6fd350b94415cde7aa142ab59a51399064dae

Reply:
> Is it possible to automatically close the flyout when the user makes a selection in the combo box? Thinking it might be better to close it once a selection has been made. What do you think?

Implemented in https://github.com/elastic/kibana/pull/155177/commits/dfc32b8e9512f57cae0b74c10fd2de8a825a4bc0